### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by dgsbl**

### CODE REVIEW

File: docs/cli-client/wasm.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
+# Wasm
+
+Wasm module allows you to manage wasm
```
Comment (zhiqiang-bianjie) [inline docs/cli-client/wasm.md:3]: Wasm module allows you to manage smart contract

File: docs/zh/cli-client/gov.md [hunk 2]
```diff
@@ -15,6 +15,7 @@
 | [tally](#iris-query-gov-tally)                  | 汇总提案投票                                          |
 | [param](#iris-query-gov-param)                  | 查询参数                                              |
 | [params](#iris-query-gov-params)                | 查询治理流程的参数                                    |
+| [proposer](#iris-query-gov-proposer)            | 查询使用给定ID提出建议的地址                                    |
```
Comment (chengwenxi) [inline docs/zh/cli-client/gov.md:18]: 查询给定ID提议的提议者

File: docs/zh/cli-client/gov.md [hunk 3]
```diff
@@ -155,6 +156,14 @@ iris query gov param [param-type] [flags]
 ```bash
 iris query gov param [param-type] [flags]
 ```
+## iris query gov proposer
+
+查询使用给定ID提出建议的地址。
```
Comment (chengwenxi) [inline docs/zh/cli-client/gov.md:161]: ditto

File: docs/cli-client/wasm.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+## iris tx wasm clear-contract-admin
+
+Clears admin for a contract to prevent further migrations
+
+```bash
+iris tx wasm clear-contract-admin [contract_addr_bech32] [flags]
+```
+
+## iris tx wasm execute
```
Comment (zhiqiang-bianjie) [inline docs/cli-client/wasm.md:30]: command should be order by `store`、`instantiate `、`execute `、`migrate `、

**Review by chengwenxi — CHANGES_REQUESTED** [review]

File: docs/zh/cli-client/gov.md (hunk 2 above)
Comment (zhiqiang-bianjie) [inline docs/zh/cli-client/gov.md:18]: 通过提议ID查询提案的发起人地址

**Review by chengwenxi — APPROVED** [review]

**Review by zhiqiang-bianjie — APPROVED** [review]

**Pull request merged**